frontend/sw360-portlet/src/main/webapp/html/components/editExternalIds.jsp

```diff
@@ -0,0 +1,63 @@
+<%@ page import="org.eclipse.sw360.portal.common.PortalConstants" %>
+<%@ page import="org.eclipse.sw360.datahandler.thrift.components.Release" %>
+<script src="<%=request.getContextPath()%>/webjars/jquery-validation/1.15.1/jquery.validate.min.js" type="text/javascript"></script>
+<script src="<%=request.getContextPath()%>/webjars/jquery-validation/1.15.1/additional-methods.min.js" type="text/javascript"></script>
+
+<table class="table info_table" id="externalIdsTable">
+    <thead>
+    <tr>
+        <th colspan="3" class="headlabel">External Ids</th>
+    </tr>
+    </thead>
+    <tbody>
+        <core_rt:forEach items="${release.externalIds.entrySet()}" var="IdsMapEntry" varStatus="loop">
+            <tr id="externalIdsTableRow${loop.count}" >
+                <td width="46%">
+                    <input id="<%=PortalConstants.EXTERNAL_ID_KEY%>${loop.count}" name="<portlet:namespace/><%=PortalConstants.EXTERNAL_ID_KEY%>${loop.count}" class="toplabelledInput" placeholder="Input name" title="Input name"
+                           value="<sw360:out value='${IdsMapEntry.key}'/>"/>
+                </td>
+                <td width="46%">
+                    <input id="<%=PortalConstants.EXTERNAL_ID_VALUE%>${loop.count}" name="<portlet:namespace/><%=PortalConstants.EXTERNAL_ID_VALUE%>${loop.count}" class="toplabelledInput" placeholder="Input id" title="Input id"
+                           value="<sw360:out value='${IdsMapEntry.value}'/>"/>
+                </td>
+                <td class="deletor">
+                    <img src="<%=request.getContextPath()%>/images/Trash.png"
+                         onclick="deleteMapItem('externalIdsTableRow${loop.count}');"
+                         alt="Delete">
+                </td>
+            </tr>
+        </core_rt:forEach>
+    </tbody>
+</table>
+
+<input type="button" class="addButton" onclick="addRowToExternalIdsTable();" value="Click to add row to External Ids "/>
+<br/>
+<br/>
+
+<script>
+
+    function deleteMapItem(rowIdOne) {
+        function deleteMapItemInternal() {
+            $('#' + rowIdOne).remove();
+        };
+
+        deleteConfirmed("Do you really want to remove this item?", deleteMapItemInternal);
+    }
+
+    function addRowToExternalIdsTable() {
+        var randomRowId = "mapItemRow" + Date.now();
+        var newRowAsString =
+            '<tr id="' + randomRowId + '" >'+
```

Comment:
Here, you basically repeat the same html code for a table row, but in a JS string. I don't like the repetition.
I see two options to get rid of it:
1. See editing of linked releases and projects. Look up usages of `linkedProjectsAjax.jsp` and go from there
2. See the new and modern way of achieving the same thing with JS and JSON in PR#530 by @lepokle: https://github.com/sw360/sw360portal/pull/530/files#diff-a2a329d1540568537c8fb67d0e245df3